src/services/sessions.js

```diff
@@ -470,6 +468,7 @@ module.exports = class SessionsHelper {
 			const processDbResponse = utils.processDbResponse(data.toJSON(), validationData)
 
 			processDbResponse['resources'] = await this.getResources(data.id, tenantCode)
```

Comment:
Why do we have to fetch resources in the creation fn since we already have the data?